scripts/backup.js

```diff
@@ -0,0 +1,35 @@
+const CronJob = require('cron').CronJob;
+ const exec = require('child_process').exec;
+require('dotenv').config('../.env');
```

Comment:
The dotenv should be called before or into the entrypoint of the server which is index.js

It should either be in `server/index.js`, or better yet, in the`package.json`, set the start script to 

`node -r dotenv/config dist/index.js`